### PR TITLE
Set light intensity for swedish

### PR DESCRIPTION
**Pull request by Robin-St**
Added intents to set light of a entity or area with tests and some updates to _common.yaml.

### CODE REVIEW

File: sentences/sv/light_HassLightSet.yaml [hunk 1]
```diff
@@ -2,4 +2,11 @@ language: sv
 intents:
   HassLightSet:
     data:
-    - sentences: []
+      - sentences:
+          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[s | n | ns ] [<ljusintensitet>] till <brightness>"
+          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[n]  till <brightness> [<ljusintensitet>]"
+          - "<name>[s | n | ns ] [upp | ner ] [till] <brightness> [<ljusintensitet>]"
+      - sentences:
+          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <alla> <ljuskällor> [<i_på>] <area>[et] [upp | ner ] till <brightness> [<ljusintensitet>]"
```
Comment (larsdunemark) [inline sentences/sv/light_HassLightSet.yaml:10]: Adding support for:
"sätt all belysning till 50% i garaget"
And maybe extract "[sätt | vrid | dra [på] | skruva | dimmra]" to 
```
expantion_rule: 
    dimma: (sätt | vrid | dra [på] | skruva | dimmra)
```
And use it as: 
```
[<dimma>] [ upp | ner] ......
```

File: sentences/sv/_common.yaml [hunk 2]
```diff
@@ -54,12 +54,15 @@ expansion_rules:
...
   slå_på: "( tänd | sätt på | slå på | starta )"
   slå_av: "( släck | stäng av | stoppa )"
-  i_på: "(i | på)"
+  i_på: "(i | på | vid | bredvid)"
   temp: "(temperatur)"
-  alla: "(all | alla )"
+  alla: "(all | alla | allt)"
+  ljusintensitet: "(ljusstyrka | intensitet | styrka | nivå | effekt)"
   lampa: "(ljuset | belysning[en] | lampan | lampor[na])"
+  ljuskällor: "(ljuskällor[na] | lampor[na] | belysning[en] | ljus[en] | glödlampor[na] | lysrör[en])"
```
Comment (larsdunemark) [inline sentences/sv/_common.yaml:62]: Should probably merge "lampa" with "ljuskällor" to use same in "TurnOn/TurnOff" as in "Set"

File: sentences/sv/light_HassLightSet.yaml (hunk 1 above)
Comment (Robin-St) [reply sentences/sv/light_HassLightSet.yaml:10]: Collecting them all under <dimma> sounds good, I will fix that. I was thinking about something similar but couldn’t decide I’m which term to collect them under :)

File: sentences/sv/_common.yaml (hunk 2 above)
Comment (Robin-St) [reply sentences/sv/_common.yaml:62]: Either that or have one ”ljuskälla” and one ”ljuskällor” to give the possibility to distinguish between singular and plural forms?

**Pull request merged**